### DISCUSSION TITLE
Overlay 'lanes' hides important street layout

### DISCUSSION THREAD

**Discussion by derkreisfahrer** [osmandapp/OsmAnd#14500] [Ideas]
**Problem** - In landscape mode the overlay "lanes" more often than not hides the most interesting part of the upcoming street-layout (see picture). 
![PROBLEM-osm-problem](https://user-images.githubusercontent.com/105595604/172723280-e3b9a4d0-f8fc-4883-9f75-fbf440209143.jpg)
Especially at inner-city-rides this can be irritating and the feature "lanes" is not as helpful as it could be. Thus, I would like to present 3 ideas:

**Idea 1** - Center own position at bottom/center not middle/center
![IDEA1-bottom_center](https://user-images.githubusercontent.com/105595604/172723346-6f18d2ea-a182-44c3-8ce1-adc92977fc6c.jpg)
If the current-postion is at the bottom/center then there is enough space for both, the upcoming street layout and the "lanes" overlay. Positioning the current-postion at the bottom/center is maybe only helpful for driving and no so much for cycling or walking. 
The setting could be named "place current position", and the values could be "middle of screen" and "bottom of screen".

**Idea 2** - Combine "turn" and "lanes"
![IDEA2-bottom_center--combine_with_turn](https://user-images.githubusercontent.com/105595604/172723404-4c45466e-ebec-499b-b764-ee637a930c80.jpg)
The overlay "lanes" and the "next turn" are combined. Thus, the overlay for "lanes" is not needed anymore. The effect for the user is the same - the immediate area in front of the current position is uncluttered, i.e. the street-layout is fully visible.
The setting could be named "Combine 'turn' and 'lanes' in one", and the values could be "yes, combine" and "no, leave them separate".

**Idea 3** - Combine "street name" and "lanes"
![IDEA3-bottom_center--combine_with_street_name](https://user-images.githubusercontent.com/105595604/172723466-13f5385b-1a94-42d2-b609-94af9aa6dbc3.jpg)
The overlay "lanes" and the "street names" are combined. Thus, the overlay for "lanes" is not needed anymore. The effect for the user is the same - the immediate area in front of the current position is uncluttered, i.e. the street-layout is fully visible.
The setting could be named "Combine 'street names' and 'lanes' in one", and the values could be "yes, combine" and "no, leave them separate".

**Comment by tommycrock** [comment]
Hi @derkreisfahrer 
Just to let you know, you can achieve idea 1 already. In the menu (3 bars) go to Settings, then select the relevant profile (such as Driving). Next choose General settings. You want to turn off "Display position always in centre". 
Now, if you have it locked on "my position" and have the map orientation set to movement direction, your position will be near the bottom.

**Reply by derkreisfahrer** [reply]
@tommycrock Thanks a lot for letting me know.
I found "Display position always in centre" and turned it off
I found "map orientation" and set it to "movement direction".
However, I did not find "my position" and (obviously) do not know how to lock it.
Maybe you can give me a hint where to find the "my position" setting or where/how to lock "my position".

**Reply by tommycrock** [reply]
For me, when I start navigation then it locks on my position. If I then pan away a blue and white target/reticle circle appear bottom right. If I click that it takes the map back to my position and holds it on it as I move. If the target has a blue background it is not locked on me, if it has a white background or is missing it is locked on me. Does that help?

By the way, I think there's probably some merit in your other ideas to not obscure the forward view so much but haven't given them enough thought to comment properly.

**Reply by derkreisfahrer** [reply]
Thanks, the explanation helps. I will definitely try that out. 

However, not having obscured the forward view so much would still be a nice feature to (optionally) have.
